monkey/infection_monkey/post_breach/pba.py

```diff
@@ -45,12 +45,17 @@ def run(self):
         """
         exec_funct = self._execute_default
         result = exec_funct()
-        hostname = socket.gethostname()
+        try:
+            hostname = socket.gethostname()
+            ip = socket.gethostbyname(hostname)
+        except socket.error:
```

Comment:
Not sure this is the right error.
socket.error is not the same as socket.gaierror which is the exception I got in my bug report.

Reply:
I know it's not the same, but gaierror, among others, should be a child to socket.error and should be caught here.

Reply:
Is it?

Reply:
I haven't set up the test env for this, that's why I'm saying "should". 
![image](https://user-images.githubusercontent.com/36815064/60412596-67e8da00-9bda-11e9-9daf-035341612a22.png)